src/pylptlib/lpt.py

```diff
@@ -1276,3 +1356,32 @@ def sweepv(instr_id: int, start_freq: float, stop_freq: float, number_of_points:
 
     err = _dll.sweepv(c_instr_id, c_start_freq, c_stop_freq, c_number_of_points, c_delay)
     check_error(err)
+
+
+"""Readout of result arrays when the measurement is started from a different command, e.g. when using List sweeps."""

```

Comment:
This docstring seems a bit lonely, there seems to be no variable or function that it describes.